docs/administration-guide/gaffer-config/schema.md

```diff
@@ -392,6 +392,16 @@ Once the schema has been loaded into a graph the parent elements are merged into
 }
 ```
 
+## Helm Deployment
```

Comment:
I'm confused as to why this is on the schema page. This page should just be about schemas. Helm config should be under Kubernetes.